Loop/Managers/LoopDataManager.swift

```diff
@@ -357,6 +447,9 @@ class LoopDataManager {
         } else {
             recommendedTempBasal = nil
         }
+
+        recommendedBolus = nil
+        recommendedBolus = try self.recommendBolus()
```

Comment:
I'm calling it to put it in the upload.  I want to be able to remotely see if I should maybe give her a bolus (if temp basal isn't enough). It's not currently displayed, but I am planning on making a change in NS to display it when hovering on the NS pill.